docs/book/usage/started.md

```diff
@@ -1,8 +1,277 @@
+have privileged access and control over what the address owns, such as tokens and objects.
+
+To view your active address, run:
+
+```bash
+sui client active-address
+```
+
+To see all your addresses and their key schemes, run:
+
+```bash
+sui client addresses
+```
+
+> **Warning: Store your keys securely**
+>
+> You must store your private key and recovery passphrase securely, otherwise you might lose
+> access to your address.
+>
+> [Learn more about addresses, available key pair options, and key storage.](https://docs.sui.io/guides/developer/getting-started/get-address)
+
+### Creating additional addresses (Optional)
+
+You can create additional addresses if needed:
+
+```bash
+sui client new-address ed25519
+```
+
+The argument `ed25519` specifies the key pair scheme to be of type `ed25519`.
+
+## Fund Sui account with tokens
+
+Before you can upload a file to Walrus and store it as a blob, you need SUI tokens to pay
+transaction fees and WAL tokens to pay for storage on the network. The Walrus Testnet uses Testnet
+WAL tokens that have no value and you can exchange them at a 1:1 rate for Testnet SUI tokens.
+
+To get SUI tokens, navigate to the SUI Testnet faucet:
+
+[https://faucet.sui.io/](https://faucet.sui.io/)
+
+Ensure you select Testnet. Then, insert your Sui address. To print your Sui address, use the
+command:
+
+```bash
+sui client active-address
+```
+
+After you insert your address on the faucet and receive a message saying you have received SUI
+tokens, check your balance with the command:
+
+```bash
+sui client balance
+```
+
+> **Tip: Faucet alternatives**
+>
+> The Sui faucet is rate limited. If you encounter errors or have questions, you can request
+> tokens from the Discord faucet or a third party faucet.
+> [Learn more about the Sui faucet.](https://docs.sui.io/guides/developer/getting-started/get-coins)
+
+Now, convert some of those SUI tokens into WAL with the command:
+
+```bash
+walrus get-wal --context testnet
+```
+
+Then, check your balance again with `sui client balance` to confirm you now have WAL:
+
+```bash
+╭─────────────────────────────────────────╮
+│ Balance of coins owned by this address  │
+├─────────────────────────────────────────┤
+│ ╭─────────────────────────────────────╮ │
+│ │ coin  balance (raw)     balance     │ │
+│ ├─────────────────────────────────────┤ │
+│ │ Sui   8869252670        0.05 SUI    │ │
+│ │ WAL   500000000         0.50 WAL    │ │
+│ ╰─────────────────────────────────────╯ │
+╰─────────────────────────────────────────╯
+```
+
+## Store a blob
+
+Changes to objects on Sui happen through the use of transactions. Accounts sign these transactions
+on behalf of addresses and they result in the system creating, updating, transferring, and
+sometimes destroying objects. Learn more about [transactions](https://docs.sui.io/concepts/transactions).
+
+To upload a file to Walrus and store it as a blob, run the following command:
+
+```bash
+walrus store file.txt --epochs 2 --context testnet
```

Comment:
Nit: Now that Testnet is set as the main context, the `--context testnet` isn't strictly necessary. However, it also doesn't hurt. 🤷